naga/tests/out/msl/spv-quad-vert.msl

```diff
@@ -12,10 +12,12 @@ struct gl_PerVertex {
     float gl_PointSize;
     type_3 gl_ClipDistance;
     type_3 gl_CullDistance;
+    char _pad4[4];
 };
 struct VertexOutput {
     metal::float2 member;
     metal::float4 gl_Position;
+    char _pad2[65519];
```

Comment:
This seems incorrect and needs further investigation

Reply:
This "shouldn't matter" was introduced in 604e10e1 (2021-4-6):

```
commit 604e10e19b1f8ce284b5da14ce72d47b670f8832
Author: Dzmitry Malyshau <kvarkus@gmail.com>
Date:   Tue Apr 6 23:40:11 2021 -0400

    Rewrite IR to have member offsets, full structure span and alignment
```

Reply:
After rebasing on 7807 this is now a sensible number